html.html

```diff
@@ -0,0 +1,168 @@
+<!--<a href="../../registro_electoral/impugnacionrenovacion.pdf" target="_blank"> DESCARGA LA PLANILLA</a>-->
```

Comment:
Please, files that are not related to the crate, if you need this content to test your code, leave it inside an string.

Reply:
Sorry it was a mistake i already deleted those files